src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -140,6 +143,20 @@ const peerConnectRequestSignChangeHandler = async (
   //TODO: Handle logic for the accept/decline sing request
 };
 
+const signifyOperationStateChangeHandler = async (
+  { record, recordType }: {record: IdentifierShortDetails, recordType: OperationPendingRecordType},
+  dispatch: ReturnType<typeof useAppDispatch>
+) => {
+  switch (recordType) {
+  case OperationPendingRecordType.IDENTIFIER:
+    dispatch(updateOrAddIdentifiersCache(record));
+    dispatch(setToastMsg(ToastMsgType.IDENTIFIER_UPDATED));
+    dispatch(setCurrentOperation(OperationType.IDLE));
```

Comment:
We shouldn't be updating the current operation, right? Doesn't have anything to do with this and this is in the background.